save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FossGraphView.kt

```diff
@@ -0,0 +1,164 @@
+/**
+ * View for FossGraph
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views
+
+import com.saveourtool.save.frontend.externals.fontawesome.faSearch
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.externals.progressbar.Color
+import com.saveourtool.save.frontend.externals.progressbar.progressBar
+import com.saveourtool.save.frontend.utils.buttonBuilder
+
+import csstype.AlignItems
+import csstype.ClassName
+import csstype.Display
+import js.core.jso
+import react.*
+import react.dom.aria.ariaDescribedBy
+import react.dom.aria.ariaLabel
+import react.dom.html.ButtonType
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.form
+import react.dom.html.ReactHTML.h1
+import react.dom.html.ReactHTML.h6
+import react.dom.html.ReactHTML.input
+import react.dom.html.ReactHTML.span
+import react.dom.html.ReactHTML.textarea
+import web.html.InputType
+
+/**
+ * [VFC] for foss graph view
+ */
+@Suppress("MAGIC_NUMBER")
+val fossGraphView: VFC = VFC {
+    val fossGraphView = fossGraphView()
+    fossGraphView {
+        name = "CVE-2022-22978"
+        progress = 87
+    }
+}
+
+/**
+ * [Props] for FossGraphView
+ */
+external interface FossGraphViewProps : Props {
+    /**
+     * Name of security vulnerabilities
+     */
+    var name: String
+
+    /**
+     * Update to change the progress and percentage
+     */
+    var progress: Int
+}
+
+@Suppress(
+    "MAGIC_NUMBER",
+    "TOO_LONG_FUNCTION",
+    "LongMethod",
+)
+private fun fossGraphView(): FC<FossGraphViewProps> = FC { props ->
```

Comment:
Suggestion to keep it `val` and maybe create a temp wrapper of type `VFC`. Just like you have now but without line `38`